_posts/2016-11-15-django14-epel6-retirement.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Django14 Retirement From EPEL 6
```

Comment:
I'm not totally satisfied with this title. I think it's too indirect to have people want to read it. Instead maybe:

`Dropping EL6 Support`
`EL6 Support Timeline`
`Upgrade your EL6 Installations`

Ideas or suggestions are welcome.